src/HttpCache/StoreAwareKernelInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Why do we need this interface and the related trait? Can't we just use inject the PSR-6 adapter in the purger?

Reply:
I can do that but to remain BC I need to introduce another interface for the Purgers that implement both `PurgerInterface` and...hmmm... `CacheTagsFormatterInterface`?

Reply:
No that's not possible. `HttpCache` works outside the application so you cannot access it from within the container.

Reply:
We could implement the same as Varnish does. Have the purger execute a `PURGE` request and have a `kernel.request` listener that purges the data. But that's way too complicated I find.